TAF_EasyRest/TestFramework/PageComponents/Owner/CreateMenuPageComponents/CreateListMenuPageComponent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using TestFramework.Pages.Owner;
+
+namespace TestFramework.PageComponents.Owner.CreateMenuPageComponents
+{
+    public class CreateListMenuPageComponent 
+    {
+        private IWebDriver driver { get; }
+        public CreateListMenuPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+  
+        private IWebElement _threeDotsButton => driver.FindElement(By.XPath("//button[@aria-label='Action']"));
+        private IWebElement _nextButton => driver.FindElement(By.XPath("//button[@aria-label='Action']/parent::div/parent::div/parent::div/parent::div/following-sibling::div[2]"));
+        private IWebElement _backButton => driver.FindElement(By.XPath("//button[@aria-label='Action']/parent::div/parent::div/parent::div/parent::div/following-sibling::div[1]"));
+
+        public CreateListMenuPageComponent ClickThreeDotsButton()
```

Comment:
Don't you need to define a new appeared element here?
![image](https://user-images.githubusercontent.com/14205615/183373805-b3415fb8-1e5d-493b-a6cd-1d60f9876e54.png)